doc/specs/#1571 - New Tab Menu Customization/#1571 - New Tab Menu Customization.md

```diff
@@ -76,6 +76,34 @@ There are five `type`s of objects in this menu:
   - The `"entries"` property specifies a list of menu entries that will appear
     nested under this entry. This can contain other `"type":"folder"` groups as
     well!
+  - The `"expand"` property accepts two values

```

Comment:
I *love* this idea. However: I read it the other way around. My brain wants to turn `expand: always` into "100% of the time, expand this folder into its parent". That is, it will insert 1..N profiles into the parent menu. Which sounds just like not having a folder at all.

We may want to rename it to `inline` or `displayInline` (`always` becomes `never`) _or_ swing the other direction and call it `displayAsSubmenu` (very explicit, `auto` and `always` still make sense)

Reply:
I like this